plugins/palo_alto_cortex_xdr/icon_palo_alto_cortex_xdr/util/api.py

```diff
@@ -419,3 +426,38 @@ def _get_endpoint_type(self, endpoint_info):
 
         # Don't know, try hostname
         return self.ENDPOINT_HOSTNAME_TYPE
+
+    def get_response_alerts(self, post_body: dict) -> Tuple[list, int, int]:
+        """
+        Helper method to make the request in `monitor_alerts` task.
+
+        :param post_body: Object containing the post body (filters etc) to send in the requests
+
+        """
+        headers = self.get_headers()
+        fqdn = self.get_url()
+        endpoint = "public_api/v1/alerts/get_alerts"
+
+        url = f"{fqdn}{endpoint}"
+
+        request = requests.Request(method="post", url=url, headers=headers, json=post_body)
+        response = make_request(_request=request, timeout=60, exception_data_location=ResponseExceptionData.RESPONSE)
+
+        response = extract_json(response)
+        total_count = response.get("reply", {}).get("total_count")
+        results_count = response.get("reply", {}).get("result_count")
+        results = response.get("reply", {}).get("alerts", [])
+
+        # They physically return None so we can't default 0 in .get()
+        if total_count is None:
+            total_count = 0
+
+        # Also including results count just to be safe.
+        if results_count is None:
+            results_count = 0
+
+        return results, results_count, total_count
+
+
+class AuthenticationError(Exception):
```

Comment:
Is this being used anywhere? could be removed if it's not

Reply:
No it's not, not sure why this is even here, I'll remove it now